frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx

```diff
@@ -26,29 +29,29 @@ type BuilderDrawerContextProps = {
   handleLogicClick: () => void
 }
 
-const BuilderDrawerContext = createContext<
-  BuilderDrawerContextProps | undefined
+const CreatePageDrawerContext = createContext<
+  CreatePageDrawerContextProps | undefined
 >(undefined)
 
 /**
  * Provider component that makes drawer context object available to any
  * child component that calls `useBuilderDrawer()`.
  */
-export const BuilderDrawerProvider: FC = ({ children }) => {
+export const CreatePageDrawerProvider: FC = ({ children }) => {
   const context = useProvideDrawerContext()
 
   return (
-    <BuilderDrawerContext.Provider value={context}>
+    <CreatePageDrawerContext.Provider value={context}>
       {children}
-    </BuilderDrawerContext.Provider>
+    </CreatePageDrawerContext.Provider>
   )
 }
 
 /**
  * Hook for components nested in ProvideAuth component to get the current auth object.
  */
-export const useBuilderDrawer = (): BuilderDrawerContextProps => {
-  const context = useContext(BuilderDrawerContext)
+export const useCreatePageDrawer = (): CreatePageDrawerContextProps => {
+  const context = useContext(CreatePageDrawerContext)
   if (!context) {
     throw new Error(
       `useBuilderDrawer must be used within a BuilderDrawerProvider component`,
```

Comment:
update this text too for consistency?

Reply:
[804d3fc](https://github.com/opengovsg/FormSG/pull/3481/commits/804d3fc36b12deceaceade133d4697bba7e1c71f)